api/src/main/resources/client_api.yaml

```diff
@@ -311,3 +362,58 @@ definitions:
   Operator:
     type: string
     enum: [EQUAL, NOT_EQUAL, LESS_THAN, GREATER_THAN, LESS_THAN_OR_EQUAL_TO, GREATER_THAN_OR_EQUAL_TO, LIKE, IN, BETWEEN]
+
+  Concept:
```

Comment:
Please add a description, this terminology is probably only familiar to OMOP users.

Reply:
Done.